config/dashboard/kano-jupyter-strings.app

```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Jupyter: Strings",
+    "tagline": "Jupyter Notebook Frontend",
+    "description": "Are you ready to interact with Python on a Browser? Hit Kano Jupyter and enjoy live coding!",
+    "slug": "jupyter",
+    "icon": "kano-jupyter-strings",
+    "colour": "#4A5152",
+    "categories": ["code"],
+    "packages": [],
+    "dependencies": ["kano-jupyter", "kano-profile"],
+    "launch_command": "kano-tracker-ctl session run 'kano-jupyter' 'kano-jupyter run --notebook=strings/Fun.ipynb'",
+    "priority": 0
```

Comment:
Likewise with `priority`, we may want to set it but probably won't worry right now.

Reply:
For the sake of simplicity I've done that too - attached screenshot to Jira ticket. Thanks!